[FEATURE] WIP Create prompt-intermediaire.json

JTEPIX opened the pull request:
Créer le module prompt intermédiaire (v1) sans LLM intégré (uniquement liens externes)

## 🌸 Problème

<!-- Décrivez ici le besoin ou l'intention couvert par cette Pull Request. -->

## 🌳 Proposition

<!-- Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés. -->

## 🐝 Remarques

<!-- Des infos supplémentaires, trucs et astuces ? -->

## 🤧 Pour tester

https://app-pr12394.review.pix.fr/modules/tmp-prompt-intermediaire/details

JTEPIX committed:
Create prompt-intermediaire.json

Créer le module prompt intermédiaire (v1) sans LLM intégré (uniquement liens externes)

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr12394.review.pix.fr)
  * [App (.org)](https://app-pr12394.review.pix.org)
  * [Orga (.fr)](https://orga-pr12394.review.pix.fr)
  * [Orga (.org)](https://orga-pr12394.review.pix.org)
  * [Certif (.fr)](https://certif-pr12394.review.pix.fr)
  * [Certif (.org)](https://certif-pr12394.review.pix.org)
  * [Junior](https://junior-pr12394.review.pix.fr)
  * [Admin](https://admin-pr12394.review.pix.fr)
  * [API](https://api-pr12394.review.pix.fr/api/)
  * [API MaDDo](https://pix-api-maddo-review-pr12394.osc-fr1.scalingo.io/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr12394.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr12394/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr12394/environment)
  * [scalingo api-maddo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr12394/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr12394/environment)

JTEPIX committed:
Update prompt-intermediaire.json

Passer isBeta en true

JTEPIX committed:
Update prompt-intermediaire.json

Quelques modifications : 
- ouvrir les liens dans un nouvel onglet
- taille des l'embed
- espaces insécables...

JTEPIX committed:
Update prompt-intermediaire.json

Modification édito suite aux retours de Jennifer

JTEPIX committed:
Update prompt-intermediaire.json

Mise en forme de la leçon avant le récap'

JTEPIX committed:
Update prompt-intermediaire.json

ajustement wording

JTEPIX committed:
Update prompt-intermediaire.json

correction de typo

JTEPIX committed:
Update prompt-intermediaire.json

modif pour ajouter image de base

JTEPIX committed:
Update prompt-intermediaire.json

Intégration des LLM et ajout des indices

JTEPIX committed:
Update prompt-intermediaire.json

Rectification URL LLM

er-lim committed:
feat(api): fix typo in prompt intermediaire module

Co-authored-by: Clément Latzarus <clement.latzarus@pix.fr>

er-lim committed:
feat(api): add tmp prefix

in prompt-intermediaire module

er-lim commented on api/src/devcomp/infrastructure/datasources/learning-content/modules/prompt-intermediaire.json:
```diff
@@ -0,0 +1,625 @@
+          "element": {
+            "id": "fbe1b91f-a423-45f5-8329-1cd50f436944",
+            "type": "text",
+            "content": "<iframe title=\"dnd\" src=\"https://epreuves.pix.fr/pix-llm/pix-llm.html?mode=cmao5mw2q000vwg014ljde4km\" height=\"800\"></iframe>"
```
Vu avec @nlepage, il faut passer par des éléments Embed pour le LLM fonctionne

er-lim committed:
feat(api): use embed element for llm

in prompt-intermediaire

Co-authored-by: Yann Bertrand <yann.bertrand@pix.fr>
Co-authored-by: Clément Latzarus <clement.latzarus@pix.fr>
